mk/config.mk

```diff
@@ -1063,3 +1063,8 @@ CFG_HMAC_64_1024_RANGE ?= n
 # By default use standard pbkdf2 implementation
 CFG_CRYPTO_HW_PBKDF2 ?= n
 $(eval $(call cfg-depends-all,CFG_CRYPTO_HW_PBKDF2,CFG_CRYPTO_PBKDF2))
+
+# Disable the widevine device tree by default
+CFG_WIDEVINE_DT ?= n
+CFG_WIDEVINE_HUK ?= $(CFG_WIDEVINE_DT)
+CFG_WIDEVINE_PTA ?= $(CFG_WIDEVINE_DT)
```

Comment:
This works, but I wonder if it wound't make sense to the order here slightly. Perhaps something like this:
```
CFG_WIDEVINE_PTA ?= n
CFG_WIDEVINE_DT ?= $(CFG_WIDEVINE_PTA)
CFG_WIDEVINE_HUK ?= $(CFG_WIDEVINE_PTA)
```